crates/dpapi/Cargo.toml

```diff
@@ -29,7 +29,12 @@ rand.workspace = true
 hmac.workspace = true
 tracing = { workspace = true, default-features = true }
 
-rust-kbkdf = { version = "1.1", git = "https://gitlab.com/TheBestTvarynka/rust-kbkdf.git", branch = "fix-key-generation" }
+kbkdf = "0.0.1"
+sha1-pre = { version = "0.11.0-pre.2", package = "sha1" }
+sha2-pre = { version = "0.11.0-pre.2", package = "sha2" }
+hmac-pre = { version = "0.13.0-pre.4", package = "hmac" }
+digest-pre = { version = "0.11.0-pre.9", package = "digest", default-features = false }
```

Comment:
The `kbkdf` crate depends on the latest versions of the `sha*` and `digest` crates. I tried older ones, but I got only compile errors about unsatisfied trait bounds.
I cannot update workspace dependencies to the latest version. I tried it too and got a lot of compilation errors in `sspi` and `picky-*` crates (I'm not willing to fix them now. let's do it in another PR)

Reply:
Oh okay… The next step would be to wait for these dependencies to go out of the prerelease phase so we can publish `kbkdf`. It’s fine for now, at least the migration work is done 👍